serverlessworkflow/modules/ROOT/pages/getting-started/cncf-serverless-workflow-specification-support.adoc

```diff
@@ -1 +1,368 @@
-= CNCF Serverless Workflow specification support
\ No newline at end of file
+CNCF Serverless Workflow Specification Support
+==============================================
+v1.0, 06-08-2022
+:compat-mode!:
+// Metadata:
+:description: CNCF Serverless Workflow Specification Support
+:keywords: kogito, workflow, serverless, cncf, specification
+// links
+:quarkus_config_guide_url: https://quarkus.io/guides/config-reference
+
+
+In this guide, you can learn the details about the Kogito implementation of the {spec_website_url}[CNCF Serverless Workflow] specification.
+
+== Current Status
+
+Currently, Kogito implements version {spec_doc_url}[{spec_version}] of the specification. The table below classifies the current implementation status for each specification feature. {spec_doc_url}#Retry-Definition[Retries] are the only feature that Kogito does not support yet.
+
+.Current implementation status of the specification features
+[cols="1,1,1"]
+|===
+| Feature | Status | Spec Reference
+
+| <<states>>
+| emoji:full_moon[]  
```

Comment:
It would be good to set a tooltip/footnote what the emojis mean? https://docs.asciidoctor.org/asciidoc/latest/macros/icon-macro/#icon-macro-attributes-shared

full moon = fully implemented and compliant with the spec
etc...

Reply:
+1

Reply:
We have a legend in the original README: https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-serverless-workflow#current-status

I'll do that instead since emojis don't have tooltip. I looked for that yesterday.